ORK1Kit/ORK1Kit/Common/ORK1FormStepViewController.m

```diff
@@ -377,6 +377,8 @@ - (void)viewWillAppear:(BOOL)animated {
     
     // Reset skipped flag - result can now be non-empty
     _skipped = NO;
+    
+    [_tableContainer layoutIfNeeded];
```

Comment:
The next button on FormStepViewController was positioned slightly lower than other step view controllers. This seems to fix it.